config/jobs/containerd/containerd/containerd-presubmit-jobs.yaml

```diff
@@ -209,8 +209,8 @@ presubmits:
         env:
         - name: USE_TEST_INFRA_LOG_DUMPING
           value: "true"
-        - name: ENABLE_CRI_SANDBOXES
-          value: "sandboxed"
+        - name: DISABLE_CRI_SANDBOXES
+          value: "true"
```

Comment:
https://github.com/containerd/containerd/blob/main/RELEASES.md?plain=1#L392, mentions that `DISABLE_CRI_SANDBOXES` should be set to `1`.   Though the code only checks the presence of environment variable, its better to have it as per the containerd docs

Reply:
Done